### PR TITLE
Remove unused variable from CI

### PR DESCRIPTION
**Pull request by marip8**
The current foxy build seems to fail because the `TMPDIR` variable is set to something specific to Gitlab:

```bash
$ docker rm 41f934a41444fb994694fcd10c3134e3cf6cd2679411b33066fc0342583e838c > /dev/null 
mktemp: failed to create file via template ‘${CI_PROJECT_DIR}.tmp/tmp.XXXXXXXXXX’: No such file or directory
/home/runner/work/_actions/ros-industrial/industrial_ci/master/industrial_ci/src/util.sh: line 414: : No such file or directory
cat: '': No such file or directory
'docker rm 41f934a41444fb994694fcd10c3134e3cf6cd2679411b33066fc0342583e838c > /dev/null ' returned with 1
```

This PR removes this variable as it is not needed for Github actions CI

### CODE REVIEW

File: .github/workflows/ubuntu.yml [hunk 1]
```diff
@@ -55,7 +55,6 @@ jobs:
           PREFIX: ${{ github.repository }}_
           DOCKER_IMAGE: ghcr.io/tesseract-robotics/tesseract_ros2:${{ matrix.distro }}-master
```
Comment (marip8) [inline .github/workflows/ubuntu.yml:56]:
```suggestion
          DOCKER_IMAGE: ghcr.io/tesseract-robotics/tesseract_ros2:${{ matrix.distro }}-0.18
```

**Comment by marrts** [comment]
This should result in actually deploying a functional docker image correct? 

If that is the case then I think we should merge it now and fix the other issues in subsequent PRs. I'm aware of a few issues beyond the vtk library that would cause the humble and rolling builds to fail, so that should be fixed in another PR.

**Comment by marip8** [comment]
> This should result in actually deploying a functional docker image correct?

Yes, I think so (at least for foxy)

> If that is the case then I think we should merge it now and fix the other issues in subsequent PRs. I'm aware of a few issues beyond the vtk library that would cause the humble and rolling builds to fail, so that should be fixed in another PR.

That's fine with me. I'm not entirely sure where the dependency on `libvtk7-dev` is being pulled in from, but I'm okay to fix in a future PR. Can you review this PR before merging?

File: .github/workflows/ubuntu.yml (hunk 1 above)
Comment (marrts) [reply .github/workflows/ubuntu.yml:56]: Looks like there currently isn't tagged versions of the docker being produced by tesseract_ros2. Also, this works with the latest version of Tesseract any way.

Comment (marip8) [reply .github/workflows/ubuntu.yml:56]: Yeah, I recently updated `tesseract_ros2` CI to build on releases, so we just need to create a tag in that repo to trigger a job that makes this docker image. I'm fine to merge this now and fix it later if you want, or we can do this first

**Comment by marrts** [comment]
All looks good to me.

**Pull request merged**